smsdk/smsdk_entities/line/line.py

```diff
@@ -0,0 +1,58 @@
+import json
+from datetime import datetime, timedelta
+from typing import List
+
+import numpy as np
+
+try:
+    import importlib.resources as pkg_resources
+except ImportError:
+    # Try backported to PY<37 `importlib_resources`.
+    import importlib_resources as pkg_resources
+
+from smsdk.tool_register import SmsdkEntities, smsdkentities
+from smsdk.utils import module_utility
+from smsdk import config
+from smsdk.ma_session import MaSession
+
+import logging
+
+log = logging.getLogger(__name__)
+
+ENDPOINTS = json.loads(pkg_resources.read_text(config, "api_endpoints.json"))
+
+
+@smsdkentities.register("line")
+class Line(SmsdkEntities, MaSession):
+    # Decorator to register a function as utility
+    # Only the registered utilites would be accessible
+    # to outside world via client.get_data()
+    mod_util = module_utility()
+
+    def __init__(self, session, base_url) -> None:
+        self.session = session
+        self.base_url = base_url
+
+    @mod_util
+    def get_lines(self, *args, **kwargs):
```

Comment:
What's the use-case for this? Do we want to use the `eb_factory` key in assets for this or `line` (which I believe is derived from `eb_factory`)?

Reply:
I'll take a look eb_factory if it has more info I'm happy to use it.  It's to get line info.